### PR TITLE
454 release notes 10.1.6

### PR DESCRIPTION
**Pull request by kirstiaa**
#454 New release 10.1.6 and new information for this release.

### CODE REVIEW

File: release-notes/10.1/core-crm/10.1.6-update.md [hunk 1]
```diff
@@ -36,10 +36,19 @@ You need the functional right **Targets administrator** for your role to add, ed
 
 Read about [Sales targets][1] on the Help Center and Community.
 
+## Company and Contact – Note with date and time stamp
```
Comment (digitaldiina) [inline release-notes/10.1/core-crm/10.1.6-update.md:39]: This must be added to core-crm/index.md also

File: release-notes/10.1/core-crm/10.1.6-update.md [hunk 2]
```diff
@@ -36,10 +36,19 @@ You need the functional right **Targets administrator** for your role to add, ed
...
+
+Using the Note tab on the main card is useful when it’s information that is important for others to see.
+
+Click the **Time stamp** button to add date, time and name to the note section, write the text section and save.
+
+![Contact (and Company) - Note tab - Add a time stamp with a button][img4]
```
Comment (digitaldiina) [inline release-notes/10.1/core-crm/10.1.6-update.md:45]: -screenshot is missing in the image tag

**Review by digitaldiina — CHANGES_REQUESTED** [review]

**Review by SuperOfficeDevNet — CHANGES_REQUESTED** [review]
Just a couple comments and suggestions. However, the comment about removing all unnecessary pronouns would suggest there are a few other changes needed to improve readability and reduce noise.

**Review by digitaldiina — APPROVED** [review]
gtg

**Pull request merged**